Alpaca.Markets/RestClient.General.cs

```diff
@@ -17,7 +17,7 @@ public sealed partial class RestClient
         public Task<IAccount> GetAccountAsync()
         {
             return getSingleObjectAsync<IAccount, JsonAccount>(
-                _alpacaHttpClient, _alpacaRestApiThrottler, "v1/account");
+                _alpacaHttpClient, _alpacaRestApiThrottler, $"v{_alpacaApiVersion}/account");
```

Comment:
Question - why not add "/v{...}" segment into BaseAddress property of _alpaceHtppClient object once?

Reply:
I like the idea, but I can't get it to work.  I put the api version on the Base Address and all the Unit tests start failing with 404 -Not Found.  The Uri is getting munged somewhere along the way and I failed to figure out why.  Maybe there was a reason it was written that way in the beginning.

Reply:
Yes, it was not easy but I prefer to do it once instead of copy-pasting this prefix each time.